### PR TITLE
UI

### PR DESCRIPTION
**Pull request by marcinwadon**
* header
* label
* button

### CODE REVIEW

File: app/ui/label/Label.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export class Label {
+  constructor(
```
Comment (mwadon) [inline app/ui/label/Label.ts]: Please add label variations: default, primary, success, info, warning, danger

Comment (marcinwadon) [inline app/ui/label/Label.ts]: Added.

File: app/ui/label/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export {LabelType} from './LabelType';
+export {Label} from './Label';
```
Comment (mwadon) [inline app/ui/label/index.ts]: Missing new line

Comment (marcinwadon) [inline app/ui/label/index.ts]: Added.

**Pull request merged**